src/module/module-collection.js

```diff
@@ -49,7 +49,20 @@ export default class ModuleCollection {
   unregister (path) {
     const parent = this.get(path.slice(0, -1))
     const key = path[path.length - 1]
-    if (!parent.getChild(key).runtime) return
+
+    if (!parent.getChild(key)) {
```

Comment:
Can we store the return value of `getChild` in a variable so that we reuse it with the next if condition?

Reply:
Oh yeah good point! Thanks. I've done it, and rebased (to the latest dev) and pushed 👍